scripts/qa_crowbarsetup.sh

```diff
@@ -1590,6 +1590,7 @@ function do_installcrowbar_cloud6plus()
 
     wait_for 60 10 "crowbar_install_status | grep -q '\"installing\": *false'" "crowbar to get installed" "tail -n 500 $crowbar_install_log ; complain 89 'crowbar install failed'"
```

Comment:
can you also change this to
`wait_for 60 10 "crowbar_install_status | grep -q '\"installing\": *false'" "Crowbar to finish installing" "tail -n 500 $crowbar_install_log ; complain 89 'Crowbar installation failed'"` ?